zerver/lib/email_notifications.py

```diff
@@ -739,6 +772,25 @@ def enqueue_welcome_emails(user: UserProfile, realm_creation: bool = False) -> N
     )
     context["getting_user_started_link"] = user.realm.uri + "/help/getting-started-with-zulip"
 
+    # We only send the onboarding_zulip_guide email for a subset of Realm.ORG_TYPES
+    onboarding_zulip_guide_url, organization_type_reference = get_onboarding_zulip_guide_context(
+        user.realm
+    )
+
+    # Only send follow_zulip_guide to "/for/communities/" guide if user is realm admin.
+    # TODO: Remove this condition when guide is updated; see https://github.com/zulip/zulip/issues/24822.
+    if (
+        onboarding_zulip_guide_url == Realm.ORG_TYPES["community"]["onboarding_zulip_guide_url"]
+        and not user.is_realm_admin
+    ):
+        onboarding_zulip_guide_url = None
+
+    if onboarding_zulip_guide_url is not None:
+        context.update(
+            organization_type=organization_type_reference,
+            zulip_guide_link=onboarding_zulip_guide_url,
+        )
```

Comment:
I think we should probably be building a separate `context` object for this second email; it seems like a recipe for some form of sneaky bug to share a Python object between the two emails here. Maybe just a `onboarding_zulip_guide_context` variable would be pretty reasonable given that we basically always reference it on a line by itself.

One question this presents is whether we want each email to have its own unsubscribe link? I think the answer is that since they don't expire, there's probably no harm in the links being identical, but it at least deserves a comment.

Reply:
I'll split out the context for this email. Sharing the unsubscribe link makes sense to me since it doesn't expire.

Do we want to clarify the text there at all, "Unsubscribe from welcome emails"? I'm thinking that we might want to add the realm name since in theory if I signed up for another Zulip Cloud organization, I wouldn't get the topics email, but I would possibly get this Zulip guide email.

Also, since this will be the third email, I'm assuming we want to separate the context objects for the `followup_day1` and `followup_day2` as well? I can do this in the revisions to the follow_up2 email.

Reply:
Let's discuss that string in chat.zulip.org; I think we might want to think about it more broadly.

And yeah, the day1/day2 emails also seem like they should have independent contexts, though we don't have to fix that to integrate this, as it's not a regression.